.rubocop.yml

```diff
@@ -63,4 +63,4 @@ Metrics/ParameterLists:
   Enabled: false
 
 Metrics/PerceivedComplexity:
-  Enabled: false
+  Enabled: false
```

Comment:
Shouldn't need to edit this file.

Reply:
True. I had disabled Style::SymbolArray then found I could use %i since bolt is Ruby 2.0+ Deleted instead of reverted but missed the newline. Will do.